### DISCUSSION TITLE
打开抖音搜索就会自动点击下面标题

### DISCUSSION THREAD

**Discussion by rjzjs** [gkd-kit/gkd#659] [Q&A]
https://github.com/gkd-kit/gkd/assets/157266132/80c46d37-9abd-4018-9701-7106c948c05b

**Comment by lisonge** [comment]
此处无法解答，你应该问提供订阅的开发者

**Reply by rjzjs** [reply]
有邮箱吗

**Reply by lisonge** [reply]
点击订阅-问题反馈
